### PR TITLE
Space Travelers' Hub: Fetch dragons data

### PR DESCRIPTION
**Pull request by helio3197**
### Added Redux action for fetching Dragons data through API.
The Redux reducer was updated adding action for fetchind the Dragons list using the Dragons API.
### Summary:
- Added fetching action function to the Redux dragon reducer file.
- Updated Dragons component for dispatching the fetch action only once when the component is mounted the first time.

### CODE REVIEW

File: src/components/Dragons.jsx [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchDragonsData } from '../redux/dragons/dragon';
 
 const Dragons = () => {
   const dragonsList = useSelector((state) => state.dragonReducer);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (dragonsList.status !== 'FETCHING_SUCCEEDED') dispatch(fetchDragonsData());
```
Comment (Saadat123456) [inline src/components/Dragons.jsx:10]: This is a good way to check if data is present in the list or not. But I think if you call `fetchDragonsData()` in the App component.

**Review by Saadat123456 — CHANGES_REQUESTED** [review]
Hello Kenny! 
You did the great job in creating actions and reducers. Also everything is working really fine but what if you call the get data function in App component because it would stay there through over the life of the application😅.

File: src/components/Dragons.jsx (hunk 1 above)
Comment (helio3197) [reply src/components/Dragons.jsx]: Thanks Saadat I've implemented the changes.

**Review by Saadat123456 — APPROVED** [review]
# Hurrayyyyyyyyy!!!!
## Changes Approved...
You did a great job @helio3197 and it's working really fine... Seeing your code is really easy to understand and you are on a really good track... Good Luck with further activities...
Cheers and Happy Coding! 🎉🎉🎉🎉🎉🎉🎉

**Pull request merged**